sgkit/distance/api.py

```diff
@@ -110,17 +123,31 @@ def pairwise_distance(
     # graph serialisation/deserialisation time significantly
     metric_param = np.empty(n_map_param, dtype=x.dtype)
 
-    def _pairwise(f: ArrayLike, g: ArrayLike) -> ArrayLike:
-        result: ArrayLike = metric_map_func(f[:, None, :], g, metric_param)
+    def _pairwise_cpu(f: ArrayLike, g: ArrayLike) -> ArrayLike:
+        # This getattr is not required when the following issues are fixed and released
```

Comment:
Can you add this as an issue please so we can make sure to address it or explicitly decide to ignore it at some point?

Reply:
Numba released a bugfix version 0.53.1, we don't need this workaround anymore.